test/feature.go

```diff
@@ -26,18 +26,24 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/cucumber/godog/gherkin"
+	"github.com/cucumber/gherkin-go/v11"
+	"github.com/cucumber/messages-go/v10"
 )
 
-func parseFeatures(filter string, paths []string) ([]*gherkin.Feature, error) {
+type feature struct {
```

Comment:
Should the type name be capitalised? Since this type will be used in other scripts, I would expect to be named "Feature" instead.

Reply:
this is used only in package

about `feature.go`, I just copied from godog source to have the possibility to retrieve the features which will be run.
I don't want to make toomuch changed yet as this will be deleted once https://github.com/cucumber/godog/issues/222 is done and released